include/firefly/panic.hpp

```diff
@@ -5,6 +5,7 @@
 
 namespace firefly {
 
+[[gnu::used]]
```

Comment:
What's this? Does this work with clang?

Reply:
Tells the compiler that this function is indeed used, otherwise it would complain about an unused function.
That was from back when we didn't panic for anything yet, I could remove it now though

Reply:
[[maybe_unused]] does the same, and is standard